src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -94,6 +94,80 @@ def provisionInfrastructure(target, item, parameters) {
   }
 }
 
+def __getRbacGroups() {
+  return [ 'developer', 'administrator' ]
+}
+
+def __getClusterRole(group) {
+  def clusterRoleMappings = [
+    developer: 'edit',
+    administrator: 'admin'
+  ]
+  return clusterRoleMappings[group]
+}
+
+def __getRole(namespace, group) {
+  return "${namespace}-${group}".toUpperCase()
+}
+
+def __getRoleArn(role) {
+  def accountId = sh(returnStdout: true, script: "aws sts get-caller-identity --query Account").replaceAll('"', '')
+  return "arn:aws:iam::${accountId}:role/${role}"
+}
+
+def __getUser(role) {
+  return "${role}-1"
```

Comment:
Does your user need a different name to the role? Or is the addition of the addition of the `-1` for readability to distinguish between the two, or just the way it has been set up in AWS land? Not a criticism, but was just wondering what the reason was.

Reply:
It's purely to distinguish it from the role. I can't imagine when we might have a '-2'.